### PR TITLE
Gift Registry management expanding

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes how to create/share/edit/delete the Gift Registry from the storefront.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-gift-registry.html

### CODE REVIEW

File: src/customers/account-dashboard-gift-registry.md [hunk 1]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
 
 ![]({% link images/images-ee/account-dashboard-gift-registry.png %}){: .zoom}
 _Gift Registry_
+
+## To create new gift registry
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:11]: Create a new gift registry

File: src/customers/account-dashboard-gift-registry.md [hunk 2]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+1. From customer account dashboard, go to **Gift Registry**.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:13]: 1. Select **Gift Registry** in the customer account dashboard.

Is that a clickable option **Gift Registry**?

File: src/customers/account-dashboard-gift-registry.md [hunk 3]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+1. On the Gift Registry page, click **Add New**.
+
+1. Select one of the **Gift Registry Type**:
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:17]: 1. Choose a **Gift Registry Type**:

File: src/customers/account-dashboard-gift-registry.md [hunk 4]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+   - Birthday
+
+   - Baby Registry
+
+   - Wedding
+
+1. Click <span class="btn">Next</span>.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:25]: 1. Click **Next**.

File: src/customers/account-dashboard-gift-registry.md [hunk 5]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+1. Fill in the required information.
+
+1. When finish, click <span class="btn">Save</span>.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:29]: 1. Fill in the required information, and click **Save**.

File: src/customers/account-dashboard-gift-registry.md [hunk 6]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+## To share gift registry
+
+1. From customer account dashboard, go to **Gift Registry**.
+
+1. Find the registry event that you want to manage, and click **Share**.
+
+1. Fill in the required information.
+
+1. When finish, click <span class="btn">Share Gift Registry</span>.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:39]: 1. Fill in the required information and click **Share Gift Registry**.

File: src/customers/account-dashboard-gift-registry.md [hunk 7]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+## To edit gift registry
+
+1. From customer account dashboard, go to **Gift Registry**.
+
+1. Find the registry event that you want to manage, and click **Edit**.
+
+1. Change any options that are necessary.
+
+1. When finish, click <span class="btn">Save</span>.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:49]: 1. Edit the required options, and click **Save**.

File: src/customers/account-dashboard-gift-registry.md [hunk 8]
```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
...
+
+## To delete gift registry
+
+1. From customer account dashboard, go to **Gift Registry**.
+
+1. Find the registry event that you want to manage, and click **Delete**.
+
+1. When prompted to confirm, click <span class="btn">OK</span>.
```
Comment (danidelcar) [inline src/customers/account-dashboard-gift-registry.md:57]: 1. Click **OK** to confirm.

**Review by danidelcar — APPROVED** [review]
Some minor changes, approved. Thanks, great work!

File: src/customers/account-dashboard-gift-registry.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: Hi @danidelcar, thank you for reviewing!

I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: Yes, this is a clickable option but not a button 😊

I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 7 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: I have implemented this suggestion!

File: src/customers/account-dashboard-gift-registry.md (hunk 8 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-gift-registry.md]: I have implemented this suggestion!

**Review by danidelcar — APPROVED** [review]
great, thanks @EvheniiaKolkhiieva !

**Comment by danidelcar** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@danidelcar the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1213`.

**Pull request merged**